docs/src/main/sphinx/admin/event-listeners-kafka.md

```diff
@@ -0,0 +1,118 @@
+# Kafka event listener
+
+The Kafka event listener plugin allows streaming of query events to an external
+Kafka-compatible system. The query history in the Kafka topic can then be
+accessed directly in Kafka, via Trino in a catalog using the [Kafka
+connector](/connector/kafka) or many downstream systems processing and storing
+the data.
+
+## Rationale
+
+This event listener is a first step to store the query history of your Trino
+cluster. The query events can provide CPU and memory usage metrics, what data is
+being accessed with resolution down to specific columns, and metadata about the
+query processing.
+
+Running the capture system separate from Trino reduces the performance impact
+and avoids downtime for non-client-facing changes.
+
+(kafka-event-listener-requirements)=
+## Requirements
+
+You need to perform the following steps:
+
+- Provide an Kafka service that is network-accessible to Trino.
+- Configure `kafka-event-listener.broker-endpoints` in the event listener
+  properties file with the URI of the service
+- Configure what events to send as detailed
+  in [](kafka-event-listener-configuration)
+
+(kafka-event-listener-configuration)=
+## Configuration
+
+To configure the Kafka event listener, create an event listener properties
+file in `etc` named `kafka-event-listener.properties` with the following
+contents as an example of a minimal required configuration:
+
+```properties
+event-listener.name=kafka-event-listener
```

Comment:
Fyi @sopel39 @marton-bod .. I consider this a bug compared to the others.. it should just be "kafka"